content/docs/1.7.0/advanced-resources/backing-image/backing-image-encryption.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Backing Image Encryption
+weight: 2
+---
+
+Starting with v1.7.0, Longhorn allows you to encrypt and decrypt a backing image by cloning it. The backing image encryption mechanism utilizes the Linux kernel module `dm_crypt` and the command-line utility `cryptsetup`.
+
+## Clone a Backing Image
+You can clone a backing image using YAML code.
```

Comment:
It is not a zero-copy clone, right? It is better to mention a cloned image also consumes disk space.

Reply:
Yes, added